src/serversideclient.h

```diff
@@ -0,0 +1,91 @@
+#pragma once
+
+#include "playerslot.h"
+#include "steam/steamclientpublic.h"
+#include "utlstring.h"
+#include "netadr.h"
+
+class CNetChan
```

Comment:
Why are we re-defining net channels here instead of using the HL2SDK? Is INetChannel broken these days?

Reply:
At the time the hl2sdk definition seemed outdated and I was still hacking around with this stuff. But now that I don't use the remote address there's no reason to keep this definition anymore. Thanks for reminding me about it.